command/policy_write.go

```diff
@@ -119,11 +120,15 @@ func (c *PolicyWriteCommand) Run(args []string) int {
 	}
 	rules := buf.String()
 
-	if err := client.Sys().PutPolicy(name, rules); err != nil {
+	if err := client.Sys().PutPolicy(formattedName, rules); err != nil {
 		c.UI.Error(fmt.Sprintf("Error uploading policy: %s", err))
 		return 2
 	}
 
-	c.UI.Output(fmt.Sprintf("Success! Uploaded policy: %s", name))
+	if name != formattedName {
+		c.UI.Warn(fmt.Sprintf("Policy name was converted to %s", formattedName))
```

Comment:
do we want to warn in the case that space was trimmed off? I'm wondering if that will look weird when warned b/c they'll just look identical haha

Reply:
The trimming is actually happening on the path to the HCL file, not the policy name. So we don't need to worry about warning because the file path name isn't getting saved anywhere, it's just getting used to populate the policy.

Reply:
Maybe I'm completely misreading the code above. But I was assuming `args[0]` is the policy name. and then we trim it to the `formattedName`

Reply:
Oh sorry I was looking at the TrimSpace on the next line for args[1]! You're right, we are trimming it. I'll change it so there's quotes around the formatted name in the warning, so the space is obvious.